.rubocop.yml

```diff
@@ -30,3 +30,7 @@ Style/IndentationConsistency:
 
 Style/RedundantReturn:
   Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
```

Comment:
Also, if we strive for consistency, this is obviously wrong, as `"` can express everything but `'` cannot. I think either style is perfectly readable and enforcing is going to be more annoying than beneficial to anyone.